docs/en/compute/Spark.md

```diff
@@ -37,19 +37,22 @@ to the focus of this guide but covered by
 [Unified and Transparent Namespace]({{ '/en/advanced/Namespace-Management.html' | relativize_url }}).
 * Make sure that the Alluxio client jar is available.
 This Alluxio client jar file can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` in the tarball
-downloaded from Alluxio [download page](https://www.alluxio.io/download).
+distribution downloaded from Alluxio [download page](https://www.alluxio.io/download).
 Alternatively, advanced users can compile this client jar from the source code
 by following the [instructions]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}).
 
 ## Basic Setup
 
-Distribute the Alluxio client jar across the nodes where Spark drivers or executors are
-running. Specifically, put the client jar on the same local path (e.g. `{{site.ALLUXIO_CLIENT_JAR_PATH}}`) on each node.
+The Alluxio client jar must be distributed across the nodes where Spark drivers or
+executors are running.
+Specifically, put the client jar on the same local path (e.g.
+`{{site.ALLUXIO_CLIENT_JAR_PATH}}`) on each node.
 
-Add the Alluxio client jar to the classpath of Spark drivers and executors
-in order for Spark applications to use the client jar to read and write files in Alluxio.
+The Alluxio client jar must be in the classpath of Spark drivers and executors
+in order for Spark applications to read and write files to Alluxio.
 Specifically, add the following line to `spark/conf/spark-defaults.conf` on every node running
-Spark. Also, make sure the client jar is copied to every node running Spark.
+Spark.
```

Comment:
nit: why wrap a line here?

Reply:
I am writing according to the documentation conventions: https://docs.alluxio.io/os/user/stable/en/contributor/Documentation-Conventions.html#line-breaks

Splitting sentences by lines makes it easier to review in diffs.
 
> Each sentence starts in a new line for ease of reviewing diffs. We do not have an official maximum characters per line for documentation files, but feel free to split sentences into separate lines to avoid needing to scroll horizontally to read.